ballerina-command/src/main/resources/cli-help/ballerina-dist.help

```diff
@@ -17,11 +17,13 @@ OPTIONS
 
 
 BALLERINA COMMANDS
-       Here is a list of available sub-commands:
-           list       List installed and available distributions
-           update     Update to the latest patch version of the active distribution
-           pull       Fetch a given distribution and set it as the active version
-           use        Make the specified distribution the active version
-           remove     Delete an installed distribution
-
-       See 'ballerina dist <command> -h' for more information on a specific command.
+        Here is a list of available subcommands:
```

Comment:
This line has move one column to the right compared to line 10 above

Reply:
Ack. will fix it.